app/models/laboratory.rb

```diff
@@ -2,9 +2,21 @@
 
 # Laboratory: represents a lab result
 class Laboratory < ApplicationRecord
+  include ValidationHelper
   belongs_to :patient
 
-  validates :result, inclusion: { in: ['positive', 'negative', 'indeterminate', 'other', nil, ''] }
+  %i[result
+     lab_type].each do |enum_field|
+    validates enum_field, on: %i[api import], inclusion: {
```

Comment:
Now that we're only applying the validations on Import and API, we should put the `result` validation back and separate it from `lab_type` which should still be API and Import only.

Reply:
Done in https://github.com/SaraAlert/SaraAlert/pull/582/commits/111784ddc5397e42ab85ecf9025f1d41b90a8ce2.